salesagility/SuiteCRM
# Fix #5115 Don't show qtip for new records

Don't show the detail tooltip when creating new Meetings/Tasks etc.

## Motivation and Context

Issue #5115 

## How To Test This


1. Go to Calendar module
2. Click on one of the days, this will pop-up a large window with many settings to create a new meeting appointment
3. Click cancel at the bottom of the pop-up
4. See there is no message box on screen, saying undefined, loading, please wait.
5. Hover over an existing Meeting and see it shows meeting details

## Types of changes


### Final checklist
